tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -3022,6 +3022,183 @@ TEST (nnstreamer_capi_singleshot, invoke_05)
 }
 #endif  /* ENABLE_NNFW_RUNTIME */
 
+#ifdef ENABLE_ARMNN
+/**
+ * @brief Test NNStreamer single shot (caffe/armnn)
+ * @detail Run pipeline with caffe lenet model.
+ */
+TEST (nnstreamer_capi_singleshot, invoke_06)
```

Comment:
When you add test cases, please add negative cases as well, which test how we behave when the input is incorrect. For each positive case, we are required to have at least one (recommended to have two or more) negative cases.

Reply:
Added.